Misc/NEWS.d/next/Core and Builtins/2022-07-18-20-47-59.gh-issue-gh-82406.IkXdhH.rst

```diff
@@ -0,0 +1 @@
+document the duck-type nature of inspect.is*function so that Cython can use it
```

Comment:
```suggestion
Document that :func:`inspect.iscoroutinefunction`, :func:`inspect.isgeneratorfunction`,
and :func:`inspect.isasyncgenfunction` now properly return ``True`` for
duck-typed function-like objects so that Cython functions can support them
```